README.md

```diff
@@ -6,6 +6,62 @@ The current set of extensions can be read on [GitHub Pages](https://ocfl.github.
 
 See also [pending pull requests](https://github.com/OCFL/extensions/pulls) for extensions under discussion.
 
+## Implementing Community Extensions
+
+Reference the OCFL specifications's description of [object extensions](https://ocfl.io/1.0/spec/#object-extensions) and [storage root extensions](https://ocfl.io/1.0/spec/#storage-root-extensions).
+
+A *root extension directory* refers to the directory named `extensions` that is located in either the storage root or an object root. An *extension directory* is an extension specific directory that is the child of a root extension directory and named using the extension's *Registered Name*. For example, `extensions/0000-example-extension` is the extension directory for the extension [0000-example-extension](docs/0000-example-extension.md).
+
+Each extension specification details how it should be implemented, but there are a few rules that apply to every extension.
+
+The OCFL storage root MAY contain a copy of an extension's specification.
+
+### Manifest
```

Comment:
Although I see the virtue in the idea of this _we-should-not-call-it-_**manifest**_-file_, I believe it is important that the content that we write to OCFL be compliant with the [1.0 specification](https://ocfl.io/1.0/spec/). With 1.0 compliance as an assumed requirement, I would suggest we keep the extension configuration files in their own extension directories.

Reply:
That is understandable. It is a shame OCFL 1.0 was released before extensions were fully realized.

I would appreciate it if you wouldn't mind adding some comments to https://github.com/OCFL/extensions/issues/34. Assuming there is no extension registry file, it remains unclear how extensions are actually supposed to work. For example, how do you know to load an extension that has no configuration? I have been trying extremely hard to be constructive and find a path forward, and am willing to continue to do so but not without constructive feedback.

Reply:
The manifest section has been removed

Reply:
:+1: